Solutions/MailGuard 365/SolutionMetadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "publisherId": "mailguardptylimited",
+    "offerId": "mailguard365",
```

Comment:
Apart from that you would need to change the `offerId `

You can refer this document  for the same - https://github.com/Azure/Azure-Sentinel/tree/master/Solutions#author-and-publish-a-solution-offer

Please repackage it after making this change.

Reply:
Hi @v-rbajaj  - I have made both the requested changes.